.github/workflows/test.yml

```diff
@@ -37,4 +37,45 @@ jobs:
         run: dotnet build --no-restore -c Debug
       - name: Test solution [Debug]
         working-directory: src
-        run: dotnet test --no-restore
+        run: dotnet test --no-restore -p:CollectCoverage=true -p:CoverletOutputFormat=lcov -p:Include=[GraphQL-Parser]* -p:CoverletOutput=../../${{ matrix.os }}.lcov.info
+      - name: Convert coverage report to clover format
+        uses: danielpalme/ReportGenerator-GitHub-Action@4.7.1
+        with:
+          reports: '${{ matrix.os }}.lcov.info'
+          targetdir: '.'
+          reporttypes: 'Clover'
+          tag: '${{ github.run_number }}_${{ github.run_id }}'
```

Comment:
How is this tag used?  Is this a required format?  Perhaps `'test_${{ github.run_number }}` would be better.  The run number will be unique for this yml file, within this repository.

Reply:
So far this tag is not used in any way. ReportGenerator was needed to just convert from one format to another: lcov -> clover. If you use ReportGenerator for a straightforward purpose - to render coverate reports in html - then this tag is displayed in the report. I've been using ReportGenerator for this on my internal projects for a long time.

Reply:
done